slack-base/src/main/java/com/hubspot/slack/client/models/commands/SlashCommandIF.java

```diff
@@ -0,0 +1,29 @@
+package com.hubspot.slack.client.models.commands;
+
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface SlashCommandIF {
+  String getToken();
+  String getTeamId();
+  String getTeamDomain();
+  Optional<String> getEnterpriseId();
+  Optional<String> getEnterpriseName();
+  String getChannelId();
+  String getChannelName();
+  String getUserId();
+  @Deprecated
```

Comment:
Why is this deprecated? Can we include a description of the reason for deprecation/link to a place to read about it?